docs/src/main/asciidoc/verifier_contract.adoc

```diff
@@ -536,6 +536,33 @@ proper values. Additionally, it registers two helper functions:
 * `escapejsonbody`: Escapes the request body in a format that can be embedded in a JSON.
 * `jsonpath`: For a given parameter, find an object in the request body.
 
```

Comment:
Hi, Marcin. I rewrote the bit that you added to make it consistent with the rest of the document. You added some good detail there. Thanks. Here's the rewrite:

==== Registering Your Own WireMock Extension

WireMock lets you register custom extensions. By default, Spring Cloud Contract registers 
the transformer, which lets you reference a request from a response. If you want to 
provide your own extensions, you can register an implementation of the
`org.springframework.cloud.contract.verifier.dsl.wiremock.WireMockExtensions` interface. 
Since we use the `spring.factories` extension approach, you can create an entry in 
`META-INF/spring.factories` file similar to the following:

[source,groovy,indent=0]
----
include::{stubrunner_core_path}/src/test/resources/META-INF/spring.factories[indent=0]
----

The following is an example of a custom extension:

.TestWireMockExtensions.groovy
[source,groovy,indent=0]
----
include::{stubrunner_core_path}/src/test/groovy/org/springframework/cloud/contract/verifier/dsl/wiremock/TestWireMockExtensions.groovy[indent=0]
----

IMPORTANT: Remember to override the `applyGlobally()` method and set it to `false` if you
want the transformation to be applied only for a mapping that explicitly requires it.